mpl_data_cast/gui/main.py

```diff
@@ -0,0 +1,235 @@
+import os
+import time
+import sys
+import traceback
+import pathlib
+import dclab
+import h5py
+import numpy
+import pkg_resources
+from PyQt6 import uic, QtCore, QtGui, QtWidgets
+
+from . import preferences
+from ..mod_recipes.rcp_rtdc import RTDCRecipe
+from .._version import version
+
+# set Qt icon theme search path
+QtGui.QIcon.setThemeSearchPaths([
+    os.path.join(pkg_resources.resource_filename("mpl_data_cast", "img"),
+                 "icon-theme")])
+QtGui.QIcon.setThemeName(".")
+
+
+class MPLDataCast(QtWidgets.QMainWindow):
+    def __init__(self, *args, **kwargs):
+        """Initialize MPL-Data-Cast"""
+        QtCore.QCoreApplication.setOrganizationName("GuckLab")
```

Comment:
I would put "MPL" here instead of GuckLab.

Reply:
It sounds so "official", that's why I hesitate to put that here. Because, legally, this is not an official software by the MPL institute, right? I mean, MPL can mean anything, so it's probably not important, but I would like to do these things the right way even for small projects like this...

Reply:
OK, ask Jochen :wink:

Reply:
The legal part is covered by the license (there is no warranty for the user) and we are MPL employees, so in principle there should not be any issue.

Reply:
Alright, I will change it.